### PR TITLE
feat: Delegation portal part 2

### PR DESCRIPTION
**Pull request by samuveth**
### Issues

Fixes https://github.com/snapshot-labs/pitches/issues/23

### Changes 

1. Add delegate votes and proposal count to the delegate card
2. Implement some of the design changes (some will be done later because they have large effect on unrelated components)
3. Add `TheActionbar` component to handle mobile action bar at the bottom of the screen
4. Implement `TheSidebar` designs (reduced size and more space for logo)
5. Add `BaseModalFullscreen` (current not used)
6. Add `BaseBreadcrumbs`  and `SpaceBreadcrumbs` currently only used for delegate profile. But ready to use for proposal space too.

### How to test

For testing with uniswap delegates:
Use the latest build and go to `testsnap.eth/delegates`

### TODOs

- [x] Add undelegate button
- [ ] ~Fix not results placeholder~ (will do in follow-up PR)
- [ ] ~Fix mobile modal to come from bottom and flexible height~ (will do in follow-up PR)
- [ ] Revert min length to 1

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [x] I have tested my changes on different screen sizes (sm, md)

### CODE REVIEW

File: src/helpers/queries.ts [hunk 1]
```diff
@@ -506,3 +506,19 @@ export const SPACE_QUERY = gql`
...
+    votes(where: { voter_in: $delegates, space: $space }) {
+      created
+      voter
+      choice
+      vp
+    }
+    proposals(where: { author_in: $delegates, space: $space }) {
+      created
+      author
+      title
```
Comment (aurelianoB) [inline src/helpers/queries.ts:521]: What's the reason for including these fields? Maybe created is enough? I know I included a few fields in the pitch but tbh it was a bit random. Don't think it has noticeable effects on performance, so no big deal anyway

File: src/composables/useDelegates.ts [hunk 2]
```diff
@@ -142,6 +149,43 @@ export function useDelegates(delegatesConfig: DelegatesConfig) {
     return tx;
   }
 
+  async function fetchDelegateVotesAndProposals(
+    delegates: string[],
+    space: string
```
Comment (ChaituVR) [inline src/composables/useDelegates.ts:154]:
```suggestion
    delegates: string[]
```

File: src/composables/useDelegates.ts [hunk 3]
```diff
@@ -142,6 +149,43 @@ export function useDelegates(delegatesConfig: DelegatesConfig) {
...
+      const delegate = vote.voter.toLowerCase();
+      votesAndProposals[delegate].votes.push(vote);
+    });
+
+    response.proposals.forEach(proposal => {
+      const delegate = proposal.author.toLowerCase();
+      votesAndProposals[delegate].proposals.push(proposal);
+    });
+
+    delegatesStats.value = votesAndProposals;
```
Comment (ChaituVR) [inline src/composables/useDelegates.ts:186]: Can we just store counts here

File: src/helpers/queries.ts (hunk 1 above)
Comment (samuveth) [reply src/helpers/queries.ts:521]: We will need these fields later when we show the detailed list of votes and proposals.

File: src/composables/useDelegates.ts (hunk 2 above)
Comment (samuveth) [reply src/composables/useDelegates.ts:154]: Space is not used currently for demo purposes

File: src/composables/useDelegates.ts (hunk 3 above)
Comment (samuveth) [reply src/composables/useDelegates.ts:186]: No, we will need the data to show the full list once we have a design

**Comment by samuveth** [comment]
Currently there is a limit set at 1k for how many proposals and votes the query will load. I think this will be enough for most cases but this might be reached for spaces that have loads of proposals. Should we increase this to 5k?

**Comment by ChaituVR** [comment]
>  Currently there is a limit set at 1k for how many proposals and votes the query will load. I think this will be enough for most cases but this might be reached for spaces that have loads of proposals. Should we increase this to 5k?

Oh no, didn't realize this. increase to 5K may increase the load on DB

**Pull request merged**